submissions/semenjob/index.html

```diff
@@ -0,0 +1,90 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+    <link rel="stylesheet" href="./style.css" />
+  </head>
+  <body>
+    <header class="header">
+      <ul>
+        <li><a href="#">Mail</a></li>
+        <li><a href="#">Images</a></li>
+      </ul>
+      <input
+        tabindex="1"
+        class="popup__checkbox"
+        type="checkbox"
+        id="popup__checkbox"
+      />
+      <label for="popup__checkbox" class="popup__checkbox__label">
+        <img src="./icon/popup-button.png" alt="" />
+      </label>
+      <nav class="popup">
+        <ul class="popup__list">
+          <li class="popup__item" tabindex="2">
+            <img class="popup__icon" src="./icon/apps.png" alt="" />
```

Comment:
`alt` shouldn't be empty. Please read why it is needed
https://www.w3schools.com/tags/att_img_alt.asp